chain/x/shard/keeper/msg_server.go

```diff
@@ -11,15 +10,18 @@ import (
 
 var _ types.MsgServer = &Keeper{}
 
-func (k *Keeper) SubmitBatch(ctx context.Context, msg *types.SubmitBatchRequest) (*types.SubmitBatchResponse, error) {
+func (k *Keeper) SubmitCardinalTx(ctx context.Context, msg *types.SubmitCardinalTxRequest,
```

Comment:
nit: SubmitShardTx. We want this to be generalizable to other shards that are not cardinal.

Reply:
done